graylog2-server/src/main/java/org/graylog2/dashboards/widgets/FieldChartWidget.java

```diff
@@ -65,15 +66,11 @@ public FieldChartWidget(MetricRegistry metricRegistry, Searches searches, String
         final ImmutableMap.Builder<String, Object> persistedConfig = ImmutableMap.<String, Object>builder()
```

Comment:
At the moment it there are no duplicated keys in widgets, so I think it's better to leave it as it is and modify it if ever needed.